integreat_cms/cms/migrations/0002_roles.py

```diff
@@ -127,9 +127,11 @@
             "delete_poi",
             "view_poi",
             "change_pushnotification",
+            "delete_pushnotification",
```

Comment:
```suggestion
```
I would not suggest to edit existing migrations.
This only works on newly setup systems and all existing databases will go into an inconsistent state (because they think they already have applied the migration, but the migration is now different)...
For more context, see [this SO answer](https://stackoverflow.com/a/37097258).